main.py

```diff
@@ -83,11 +104,20 @@ def run(self):
             if PREF.get('distraction_free_window.toggle_status_bar', False):
                 w.set_status_bar_visible(True)
 
-        try:
-            # toggle MaxPane if found
-            PKGCTRL_PREF = sublime.load_settings('Package Control.sublime-settings')
-            is_maxpane_installed = bool('MaxPane' in set(PKGCTRL_PREF.get('installed_packages', [])))
-            if is_maxpane_installed:
-                self.window.run_command('max_pane')
-        except Exception as e:
-            pass
+        if toggle:
+            try:
+                # toggle MaxPane if found
+                PKGCTRL_PREF = sublime.load_settings('Package Control.sublime-settings')
+                is_maxpane_installed = bool('MaxPane' in set(PKGCTRL_PREF.get('installed_packages', [])))
+                if is_maxpane_installed:
+                    self.window.run_command('max_pane')
+            except Exception as e:
+                pass
+
+            try:
+                with open(session_file, "w") as file:
```

Comment:
```python
with os.open(session_file, mode='w', newline='\n') as file:
```

Reply:
I think you mean open() instead of os.open(), else this error is raised:
TypeError: Required argument 'flags' (pos 2) not found

Added the newline param in open(), thank you.